signing_service/signing_service/signing_service.py

```diff
@@ -81,7 +81,7 @@ class SigningService:
         'maximum_reconnection_attempts',
         'notifier',
         'signing_service_daily_transaction_sum_so_far',
-        'last_signed_transaction',
+        'last_transactions_limit_file',
```

Comment:
why was it renamed? last name was better

Reply:
Because of comment above:
>This name is misleading, it is suggesting that you will have a ethereum.Tranaction object here, not a date string.

I do agree with @rwrzesien , it's just a name of the file we are storing.

Reply:
Yea, the previous name suggested it is a `Transaction` object. But the new name suggest it is `file` object ;) I would add `last_transactions_limit_file_name`.

Reply:
or simpler - `last_signed_transaction_date`?